host-configs/apple/macOS_base.cmake

```diff
@@ -6,24 +6,38 @@ message("CONFIG_NAME = ${CONFIG_NAME}")
 
 set(CMAKE_C_COMPILER "/usr/bin/clang" CACHE PATH "")
 set(CMAKE_CXX_COMPILER "/usr/bin/clang++" CACHE PATH "")
+set(CMAKE_FORTRAN_COMPILER "/usr/bin/gfortra`" CACHE PATH "")
 set(ENABLE_FORTRAN OFF CACHE BOOL "" FORCE)
 
 set(ENABLE_MPI ON CACHE PATH "")
-set(MPI_C_COMPILER "${HOMEBREW_DIR}/bin/mpicc" CACHE PATH "")
-set(MPI_CXX_COMPILER "${HOMEBREW_DIR}/bin/mpicxx" CACHE PATH "")
-set(MPIEXEC "${HOMEBREW_DIR}/bin/mpirun" CACHE PATH "")
+#set(MPI_C_COMPILER "${HOMEBREW_DIR}/bin/mpicc" CACHE PATH "")
+#set(MPI_CXX_COMPILER "${HOMEBREW_DIR}/bin/mpicxx" CACHE PATH "")
+#set(MPI_Fortran_COMPILER "${HOMEBREW_DIR}/bin/mpifort" CACHE PATH "")
+#set(MPIEXEC "${HOMEBREW_DIR}/bin/mpirun" CACHE PATH "")
+
+set(MPI_C_COMPILER "mpicc" CACHE PATH "")
+set(MPI_CXX_COMPILER "mpicxx" CACHE PATH "")
+set(MPI_Fortran_COMPILER "mpifort" CACHE PATH "")
+set(MPIEXEC "mpirun" CACHE PATH "")
 
 set(ENABLE_GTEST_DEATH_TESTS ON CACHE BOOL "" FORCE)
 
 set(ENABLE_PVTPackage ON CACHE BOOL "" FORCE)
 
 set(ENABLE_CUDA "OFF" CACHE PATH "" FORCE)
 set(ENABLE_OPENMP "OFF" CACHE PATH "" FORCE)
-
 set(ENABLE_CALIPER "OFF" CACHE PATH "" FORCE )
+set(ENABLE_TRILINOS "OFF" CACHE PATH "" FORCE )
+set(ENABLE_PETSC "OFF" CACHE PATH "" FORCE )
+set(ENABLE_SCOTCH "OFF" CACHE PATH "" FORCE )
 
-set( BLAS_LIBRARIES ${HOMEBREW_DIR}/opt/lapack/lib/libblas.dylib CACHE PATH "" FORCE )
-set( LAPACK_LIBRARIES ${HOMEBREW_DIR}/opt/lapack/lib/liblapack.dylib CACHE PATH "" FORCE )
+#set( BLAS_LIBRARIES ${HOMEBREW_DIR}/opt/lapack/lib/libblas.dylib CACHE PATH "" FORCE )
+#set( LAPACK_LIBRARIES ${HOMEBREW_DIR}/opt/lapack/lib/liblapack.dylib CACHE PATH "" FORCE )
+
+#set( BLAS_LIBRARIES /Users/byer3/apps/opt/lapack-3.11/libblas.3.dylib CACHE PATH "" FORCE )
+#set( LAPACK_LIBRARIES /Users/byer3/apps/opt/lapack-3.11/liblapack.3.dylib CACHE PATH "" FORCE )
+set( BLAS_LIBRARIES /Users/byer3/apps/opt/lapack-3.11/libblas.dylib CACHE PATH "" FORCE )
+set( LAPACK_LIBRARIES /Users/byer3/apps/opt/lapack-3.11/liblapack.dylib CACHE PATH "" FORCE )
```

Comment:
same for these since they only work for your machine.

Reply:
done